utils/helpers.ts

```
@@ -207,15 +207,29 @@ export function calculateAverageTimeToBlock(hashRate: bigint | number | string,
  * @returns {object} An object with keys '1h', '1d', '1w', '1m', '1y' mapping to probability strings (e.g., "0.123%", "<0.001%")
  */
-export function calculateBlockChances(hashRate: bigint, difficulty: number, accepted: bigint): { [key: string]: string} {
-  const difficultyFactor = Math.round(Number(difficulty) * 100);
-  const acceptedBigInt = typeof accepted === 'bigint' ? accepted : BigInt(accepted);
-  
-  if (!Number.isFinite(difficultyFactor) || difficultyFactor <= 0 || acceptedBigInt <= BigInt(0)) {
+export function calculateBlockChances(
+  hashRate: bigint | number | string,
+  difficulty: number | bigint,
+  accepted: bigint | number | string
+): { [key: string]: string} {
```

ESLint

[error] Insert `·`

(prettier/prettier)